src/sidebar/services/frame-sync.ts

```diff
@@ -1,4 +1,5 @@
 import debounce from 'lodash.debounce';
+import type { DebouncedFunction } from 'lodash.debounce';
```

Comment:
This makes me wonder if we should adopt this debounce pattern more consistently (cf. e.g. poor-man's trailing-edge "scheduling" using `setTimeout`). Thoughts?

Reply:
In cases where the behavior we need is easily expressed with `debounce` and its options, it makes sense to use that. I think the main reason to avoid it would be if you needed some flexibility it didn't provide (eg. different scheduling mechanism than `setTimeout`), or we're writing code in some context where we wanted to avoid the dependency (not applicable anywhere in the Hypothesis client yet, but perhaps in a shared library in future).